### ISSUE TITLE
Tests fail since I updated my crates

### ISSUE THREAD

**Issue by dns2utf8** [gotham-rs/gotham#523]
Hi all

I noticed that my test currently fail with this error:
```
panicked at 'not currently running on a Tokio 0.2.x runtime.'
```

I see in the git repo that the tokio runtime has been updated to `1.x` already.
Will there be a new release of gotham anytime soon?

Cheers,
Stefan

**Comment by msrd0** [comment]
The latest release (0.5.0) is only compatible with tokio 0.2.x, so you need to use the git version of gotham if you want to use tokio 1.0 today.

Given that there is very little activity on the issues on the `0.6.0` milestone, it might make sense to make a smaller release soon instead of waiting until someone gets around to fixing those issues. @nyarly @sezna what do you think?

**Comment by dns2utf8** [comment]
An `0.6.0` release would be great. I use my server to query other https servers with reqwest, which in the lastes version `0.11.x` uses tokio `1.2.x` too.

**Comment by msrd0** [comment]
I just released version 0.6.0 so gotham is now compatible with tokio 1.0
